rubicon-htb.js

```diff
@@ -755,7 +787,7 @@ function RubiconModule(configs) {
          * ---------------------------------- */
 
         //? if (TEST) {
-        __profile: __profile,
+        profile: __profile,
```

Comment:
Please change this back.

Reply:
Yes but the test file is using: partnerModule.profile

Reply:
Can you please for now change the test file to use the variable with __ and change back the key? We will investigate the spec file issue on our end in the meantime.